cypress/integration/rules_spec.js

```diff
@@ -122,8 +122,16 @@ describe('Rules page', () => {
       cy.contains('.heading--large', 'Copy a rule');
 
       const newName = 'testRule2';
+      cy.contains('.ace_variable', 'name');
```

Comment:
My thinking is that since 'name' will be present for the default empty rule object and the copied rule, and the copied rule will only be present after a rerender, we need to look for something specific to the copied rule.